subprojects/jacoco/src/main/java/org/gradle/testing/jacoco/plugins/JacocoReportAggregationPlugin.java

```diff
@@ -68,68 +68,70 @@ public void apply(Project project) {
         project.getPluginManager().apply("jacoco");
 
         RoleBasedConfigurationContainerInternal configurations = ((ProjectInternal) project).getConfigurations();
-        Configuration jacocoAggregation = configurations.bucket(JACOCO_AGGREGATION_CONFIGURATION_NAME);
-        jacocoAggregation.setDescription("Collects project dependencies for purposes of JaCoCo coverage report aggregation");
-        jacocoAggregation.setVisible(false);
-        jacocoAggregation.setTransitive(true);
+        NamedDomainObjectProvider<Configuration> jacocoAggregation = configurations.dependenciesUnlocked(JACOCO_AGGREGATION_CONFIGURATION_NAME, conf -> {
```

Comment:
This could potentially be a locked `DependenciesConfiguration` because this is incubating.